### ISSUE TITLE
player:SendListInventory does not fire

### ISSUE THREAD

**Issue by PsYchoNauTek** [azerothcore/mod-eluna#109]
want to do a multi vendor script via eluna... so wanted to use the player:SendListInventory function but nothing happens at all.

**Comment by 55Honey** [comment]
I expect this to fire an error, since you're passing 2 arguments for a method, which expects only one.
See
https://www.azerothcore.org/pages/eluna/Player/SendListInventory.html

**Comment by PsYchoNauTek** [comment]
yeah but if u look in the playermethods.h at that function it is different and according to that i coded my part 

here u see:


    int SendListInventory(lua_State* L, Player* player)
    {
        WorldObject* obj = Eluna::CHECKOBJ<WorldObject>(L, 2);
        uint32 vendorId = Eluna::CHECKVAL<uint32>(L, 3, 0);

        player->GetSession()->SendListInventory(obj->GET_GUID(), vendorId);
        return 0;
    }

**Comment by PsYchoNauTek** [comment]
since the way i understant it is that it takes the creature which we are talking to (so obj->GET_GUID()) and then state the vendor entry which we want to send to player (so the vendorId)

**Comment by 55Honey** [comment]
See https://github.com/azerothcore/mod-eluna/pull/48 for an example and what table to pick numbers from.

Also we need to add the param vendorId to the doxygen, which was added in PR 48.
Obviously we missed it when reviewing the PR.

**Comment by PsYchoNauTek** [comment]
Yeah from codestyle it looks quite the same like on noisiver ^^


Ah kk I see I see =)
Good then I am looking forward to the fully implemented version of it! =D

**Comment by 55Honey** [comment]
Since noisiver got it working, i am not sure what is missing. Did you try a custom vendor id? Try one from the core?

**Comment by PsYchoNauTek** [comment]
OMFG... I was just an idiot!

From script all was fine...
But I forgot to implement on the updated fresh compiled server at lua_scripts AIO_Server 🤦‍♂️ x100
So yeah... that was the problem in the end... works now fine with regular blizz vendors as well as custom ones... xD ^^''

**Comment by 55Honey** [comment]
I'm glad you solved it!
